pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>org.icepear.echarts</groupId>
   <artifactId>echarts-java</artifactId>
-  <version>1.0.2</version>
+  <version>1.0.3</version>
```

Comment:
be reminded to change the version in reademe and docs as well

Reply:
revert changes on version since it is not ready for release